LICENSE

```diff
@@ -310,12 +306,16 @@
     python/mxnet/onnx/mx2onnx/_export_onnx.py
     python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset12.py
     python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset13.py
+    src/operator/numpy/np_einsum_op-inl.h
```

Comment:
should we list functions of BSD clause.

Reply:
Based on the history of the file, it seems as if it was heavily modified, so I think we are correct to release it under ASF 2.0 license and don't need to call out specific functions.

Reply:
We also don't call out specific functions in other files in this LICENSE category.